sklearn/manifold/_barnes_hut_tsne.pyx

```diff
@@ -53,17 +55,18 @@ cdef float compute_gradient(float[:] val_P,
                             int dof,
                             long start,
                             long stop,
-                            bint compute_error) nogil:
+                            bint compute_error,
+                            int num_threads) nogil:
```

Comment:
Nit: our convention is to use `n_threads`

Reply:
The argument in the `prange` is called `num_threads` and this is private function. I would rather keep it that way as it seems clearer to me (it is linked to `*_NUM_THREADS` and all the openMP naming) but if you strongly disagree, I will make the change.